hook.php

```diff
@@ -437,3 +443,56 @@ function plugin_formcreator_dynamicReport($params) {
 
    return false;
 }
+
+/**
+ * Delete all form answers for a given form
+ * Called by GLPI when a form is about to be purged
+ *
+ * @param CommonDBTM $item
+ */
+function plugin_formcreator_hook_purge_pluginformcreatorform(CommonDBTM $item) {
+   if ($item instanceof PluginFormcreatorForm) {
+      $formAnswer = new PluginFormcreatorFormAnswer();
+      $success = $formAnswer->deleteByCriteria([
+         PluginFormcreatorForm::getForeignKeyField() => $this->getID(),
+      ]);
+
+      if (!$success) {
+         // Failed to delete linked items
+         $item->input = false;
+      }
+   }
+}
```

Comment:
thus function is currently useless because of the current design (see my comment about the PR). It will be useful if we choose to do cascaded deletion.